packages/graph/src/graph2/graph.ts

```diff
@@ -353,6 +353,9 @@ export class Graph2 extends SVGZoomWidget {
         }
         if (x !== undefined && y !== undefined) {
             const bbox = item.element.node().getBBox();
+            const proj = this.projectPlacholder(item as VertexPlaceholder);
```

Comment:
Forcing to a VertexPlaceholder is dangerous, put it in a `this._graphData.isVertex(item)` and you can remove the "as"

Reply:
(item might also be a SubgraphPlaceholder or an EdgePlaceholder) so you may need to rework the logic a little bit...

Reply:
I suspect this code should be relocated to line 348->350